### PR TITLE
Dimension Nodes 2016

### PR DESCRIPTION
**Pull request by moethu**
### Purpose

Nodes for Revit Dimensions added

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers

@sharadkjaiswal 

### FYIs

@Randy-Ma , @mjkkirschner

### CODE REVIEW

File: src/Libraries/RevitNodes/Elements/Dimension.cs [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+        /// Init from existing
+        /// </summary>
+        /// <param name="element"></param>
+        private void InitElement(Autodesk.Revit.DB.Dimension element)
+        {
+            InternalSetElement(element);
+        }
+
+        /// <summary>
+        /// Init by By Elements
```
Comment (mjkkirschner) [inline src/Libraries/RevitNodes/Elements/Dimension.cs]: typo

File: src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs [hunk 2]
```diff
@@ -212,9 +212,15 @@ public static Mullion Wrap(Autodesk.Revit.DB.Mullion ele, bool isRevitOwned)
            return Mullion.FromExisting(ele, isRevitOwned);
         }
 
+<<<<<<< HEAD
+        public static Dimension Wrap(Autodesk.Revit.DB.Dimension ele, bool isRevitOwned)
```
Comment (mjkkirschner) [inline src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs]: conflict?

File: test/Libraries/RevitNodesTests/Elements/DimensionTests.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+using NUnit.Framework;
+
+using RevitTestServices;
+
+using RTF.Framework;
+
+namespace RevitNodesTests.Elements
+{
+    [TestFixture]
+    public class DimensionTests : RevitNodeTestBase
```
Comment (mjkkirschner) [inline test/Libraries/RevitNodesTests/Elements/DimensionTests.cs]: can you also add a system test that loads a .dyn here?

**Comment by moethu** [comment]
@mjkkirschner issues resolved. System tests added

**Comment by mjkkirschner** [comment]
@moethu this looks good to go, should we merge this next?

**Comment by moethu** [comment]
yes now it should be fine. Just merged the latest changes from 2016 into this branch.

**Pull request merged**